cli/testflinger_cli/__init__.py

```diff
@@ -425,6 +441,18 @@ def agent_status(self):
 
     def queue_status(self):
         """Show the status of the agents in a specified queue."""
+        if self.args.jobs:
+            # New job-focused functionality
+            self._queue_status_jobs()
+        else:
+            # Original agent-focused functionality
+            self._queue_status_agents()
+
+    def _queue_status_agents(self):
+        """Show the status of the agents in a specified queue.
+
+        This is the original functionality.
```

Comment:
This can be reformatted as follow based on current suggestions:
```suggestion
    def _queue_status_agents(self):
        """Retrieve the status of the agents in a specified queue.
```